lib/blockchain.py

```diff
@@ -193,7 +193,7 @@ def verify_header(self, header, prev_header, bits):
         if prev_hash != header.get('prev_block_hash'):
             raise VerifyError("prev hash mismatch: %s vs %s" % (prev_hash, header.get('prev_block_hash')))
         # checkpoint BitcoinCash fork block
-        if ( header.get('block_height') == BITCOIN_CASH_FORK_BLOCK_HEIGHT and hash_header(header) != BITCOIN_CASH_FORK_BLOCK_HASH ):
```

Comment:
These constants should be moved to NetworkConstants; that's the real issue here

Reply:
Good point!